PayrollApplicationGUI.java

```diff
@@ -166,7 +177,26 @@ private double computeTax(double gs) {
         return 0.0;
     }
 
+    private void setLabelText(JLabel label, String text) {
+        label.setText(label.getText() + " " + text);
+    }
+
+    private String currencyFormat(double val) {
```

Comment:
by default the amount can be in rupee instead of dollar. Or we can create a option if user wants the amount in USD or INR